roc/sender_test.go

```diff
@@ -82,6 +83,62 @@ func TestSender_Open(t *testing.T) {
 	}
 }
 
+func TestSender_SetOutgoingAddress(t *testing.T) {
+	cases := []struct {
+		name    string
+		slot    Slot
+		iface   Interface
+		ip      string
+		wantErr error
+	}{
+		{
+			name:    "ok",
+			slot:    SlotDefault,
+			iface:   InterfaceAudioSource,
+			ip:      "127.0.0.1",
+			wantErr: nil,
+		},
+		{
+			name:    "bad iface",
+			slot:    SlotDefault,
+			iface:   -1,
+			ip:      "127.0.0.1",
+			wantErr: newNativeErr("roc_sender_set_outgoing_address()", -1),
+		},
+		{
+			name:    "invalid ip",
+			slot:    SlotDefault,
+			iface:   InterfaceAudioSource,
+			ip:      "127.0.0.1\x00",
+			wantErr: fmt.Errorf("invalid ip: unexpected zero byte in the string: \"127.0.0.1\\x00\""),
+		},
```

Comment:
It would be also nice to add a similar case with invalid IP, but when validation fails on the C side (e.g. something line "256.256.256.256")

Reply:
done